proto/mls/api/v1/mls.proto

```diff
@@ -78,15 +78,47 @@ service MlsApi {
       body: "*"
     };
   }
+
+  // Query stored group messages
+  rpc QueryGroupMessages(QueryGroupMessagesRequest) returns (QueryGroupMessagesResponse) {
+    option (google.api.http) = {
+      post: "/mls/v1/query-group-messages"
+      body: "*"
+    };
+  }
+
+  // Query stored group messages
+  rpc QueryWelcomeMessages(QueryWelcomeMessagesRequest) returns (QueryWelcomeMessagesResponse) {
+    option (google.api.http) = {
+      post: "/mls/v1/query-welcome-messages"
+      body: "*"
+    };
+  }
+
+  // Follow stream of new group messages
+  rpc FollowGroupMessages(FollowGroupMessagesRequest) returns (stream xmtp.mls.message_contents.GroupMessage) {
```

Comment:
Why follow instead of subscribe? Are any semantics different, or do we just think it's a better name?

Reply:
Yea same semantics just different name

Reply:
Changed this back to Subscribe per our conversation IRL